### ISSUE TITLE
Abnormal sperm function

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9842]
**Preferred term label:**
Abnormal sperm function

**Synonyms**

**Definition (free text, please give PubMed ID)**
In men with abnormal sperm function the underlying cause of the couple’s infertility often remains undiagnosed, or the diagnosis is significantly delayed because of the normal appearance of the sperm and semen. Sperm functions including protection of paternal DNA, traversing the female reproductive tract, oocyte localisation, penetration of the zona pellucida, oocyte activation and centriole deposition can be impaired without resulting in impaired motility or abnormal morphology. 

**Parent term (use hpo.jax.org/app)**
HP:0012041

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 98048, ORPHA: 399980

**Your nano-attribution (ORCID)**
0000-0002-7651-2403

**Comment by pnrobinson** [comment]
I think that this is better placed as a child term of
[Functional abnormality of male internal genitalia](https://hpo.jax.org/app/browse/term/HP:0040323)

Also, to make the label be consistent with other HPO terms, I will make the label be
Abnormal sperm physiology
and put Abnormal sperm function as a synonym.

added as HP:0034809

**Comment by MagoWyr** [comment]
I am not sure whether the sperm function should be part of the male internal genitalia. Most of the sperm functions are required for fertilising the oocyte, which happens inside the female genital tract.

But I am fine with calling it Abnormal sperm physiology.
________________________________
